### PR TITLE
Add warning message for Danger zone in case of editing the Topic #2534

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Added warning message for Danger zone in case of editing the Topic
 
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/theme/theme.ts [hunk 1]
```diff
@@ -57,6 +57,9 @@ const Colors = {
     '45': '#5865F2',
     '50': '#5B67E3',
   },
+  gray: {
+    '50': '#73848C',
+  },
```
Comment (workshur) [inline kafka-ui-react-app/src/theme/theme.ts:62]: pls use existing palete

**Review by workshur — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/theme/theme.ts (hunk 1 above)
Comment (David-DB88) [reply kafka-ui-react-app/src/theme/theme.ts]: done

**Review by workshur — APPROVED** [review]
@Haarolean pls take a look 👀

File: kafka-ui-react-app/src/components/Topics/Topic/Edit/DangerZone/DangerZone.tsx [hunk 2]
```diff
@@ -81,6 +81,10 @@ const DangerZone: React.FC<DangerZoneProps> = ({
   return (
     <S.Wrapper>
       <S.Title>Danger Zone</S.Title>
+      <S.Warning>
+        Change these parameters only if you are absolutely sure that you are
```
Comment (Haarolean) [inline kafka-ui-react-app/src/components/Topics/Topic/Edit/DangerZone/DangerZone.tsx:85]:
```suggestion
        Change these parameters only if you are absolutely sure what you are
```

**Review by Haarolean — CHANGES_REQUESTED** [review]

**Review by Haarolean — APPROVED** [review]

**Pull request merged**